dvc/tree/azure.py

```diff
@@ -170,6 +170,14 @@ def getsize(self, path_info):
     def get_file_hash(self, path_info):
         return HashInfo(self.PARAM_CHECKSUM, self.get_etag(path_info))
 
+    def _upload_fobj(self, fobj, to_info):
+        blob_client = self.blob_service.get_blob_client(
+            to_info.bucket, to_info.path
+        )
+        blob_client.max_block_size = self.CHUNK_SIZE
+        blob_client.min_large_block_upload_threshold = self.CHUNK_SIZE
```

Comment:
Btw, this might result in timeout errors. Maybe let's just let it use default chunk size? IIRC s3 (and maybe azure, but in gs we do this ourselves, see dynamic_chunk_size) tries to adjust chunk size until it stops running into timeout errors.

Reply:
Same question for the rest of remotes.

Also, if we use defaults here, I guess we could reuse `upload_fobj` in `_upload` down below.

Reply:
> Same question for the rest of remotes.

What do you suggest? Lowering it down to something like 8MB or just removing it from azure/s3?

Reply:
@isidentical I think we can just not use `CHUNK_SIZE` in s3 (possibly others) to let them figure out the best chunk size themselves. Unless I'm missing something.

Reply:
Alright, I'll strip the `CHUNK_SIZE` from all providers tomorrow (in order to first test under heavy conditions to see whether we would experience memory issues or not)